launcher/src/se/llbit/chunky/launcher/JavaFxLocator.java

```diff
@@ -61,16 +156,19 @@ private static void runWithJavafx(Path javafxDir, String[] args) {
     cmd.add("--javaOptions");
     StringBuilder javaOptions = new StringBuilder();
     javaOptions.append("--module-path ");
-    if (System.getProperty("os.name").startsWith("Windows")) {
-      // Escape the path twice to make the second launcher pass the options to Chunky retaining the double speechmarks (fixes paths with spaces)
-      javaOptions.append("\\\"" + javafxDir.toAbsolutePath().toString() + "\\\"");
+    if(System.getProperty("os.name").startsWith("Windows")) {
+      // Escape the path twice to make the second launcher pass the options to Chunky retaining the double quotation marks (fixes paths with spaces)
+      javaOptions.append("\\\"").append(javafxDir.toAbsolutePath()).append("\\\"");
     } else {
-      javaOptions.append(javafxDir.toAbsolutePath().toString());
+      javaOptions.append(javafxDir);
     }
     javaOptions.append(" --add-modules ");
     javaOptions.append("javafx.controls,javafx.fxml");
     cmd.add(javaOptions.toString());
 
+    System.out.println("Trying to start the chunky process with the following arguments:");
+    System.out.println(String.join(" ", cmd));
```

Comment:
Maybe change this to use the logger?

Reply:
ChunkyLauncher.java (which calls JavaFxLocator) does not use loggers, only println. A logger is initially created by ChunkyDeployer::launchChunky using the given loggerBuilder.